app/frontend/src/setupTests.ts

```diff
@@ -8,7 +8,42 @@ import mediaQuery from "css-mediaquery";
 
 import user from "./test/fixtures/defaultUser.json";
 
-jest.mock("./service");
+const mockedService: any = {};
+
+jest.mock("./service", () => ({
```

Comment:
I added this and "fixed" three instances where api calls weren't being mocked, but it didn't make any difference and I'm not sure if we should actually keep this.

Reply:
Probably not, I remember it made `console.log`ing to figure out jest mock specific things a pain in the arse since all you get is `Proxy` in the output, and jest generates a mock stub automatically anyway (sort of, but with no implementation) with the code before the change.

Reply:
Actually, the pain might not be a problem here, but still, probably shouldn't write more code than needed :p